unstructured/staging/label_box.py

```diff
@@ -0,0 +1,84 @@
+import os
+
+from typing import Any, Dict, List, Optional
+from unstructured.documents.elements import Text
+
+
+VALID_ATTACHMENT_TYPES = ["IMAGE", "VIDEO", "RAW_TEXT", "TEXT_URL", "HTML"]
+
+
+def _validate_attachments(attachments, element_index):
+    for attachment_index, attachment in enumerate(attachments):
+        error_message_prefix = f"Error at index {attachment_index} of attachments parameter for element at index {element_index}."
```

Comment:
This line is too long and triggering the line length error in the lint job. Max width is 100. I've been formatting long strings like this:

```python
error_message_prefix = f"Error at index {attachment_index} of attachments "
                                          f"parameter for element at index {element_index}."
```